### PR TITLE
Add documentation about how to resize azure managed disk

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/jira/software/c/projects/DLT/issues/DLT-7172

# Done
Create documentation

### CODE REVIEW

File: docs/HowToExpandAzureManagedDisk.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Resize the disk
+
+Follow [the guide](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/expand-disks#expand-an-azure-managed-disk)
+
+## Start the VM (or Attach the disk)
+
+Follow [the guide](https://docs.microsoft.com/en-us/azure/devtest-labs/devtest-lab-attach-detach-data-disk#attach-a-data-disk)
+
+Note that the `LUN` must be chosen with appropriate value when using `Atttach disk`
```
Comment (ymorimo) [inline docs/HowToExpandAzureManagedDisk.md:27]:
```suggestion
Note that the `LUN` must be chosen with appropriate value when using `Attach disk`
```

File: docs/HowToExpandAzureManagedDisk.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Since `scalar-terraform` manages a set of nodes in a cluster (e.g. Cassandra cluster) in the same way, it is not allowed to change the resource configuration of only part of the nodes. So please update all the nodes in the same cluster in the same way.
+For example, if you update the volume size of a Cassandra node from 1TB to 2TB, you need to update the volumes of the other Cassandra nodes to 2TB as well.
+
+## Update tfstate
+
+Since the above operations are done manually without using terraform, the actual states of cloud resources and tfstate are not consistent after the above steps. Please do the following to update tfstate to make it consistent with the actual resources.
+
+First, update the tfvars of a terraform module of a resource that you updated to match with the actual states of the resource.
+For example, if you update the size of a data volume of Casssandra node to 2TB, please do as follows.
```
Comment (ymorimo) [inline docs/HowToExpandAzureManagedDisk.md:61]:
```suggestion
For example, if you update the size of a data volume of Cassandra node to 2TB, please do as follows.
```

**Review by ymorimo — APPROVED** [review]
LGTM except for some misspellings

File: docs/HowToExpandAzureManagedDisk.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# A Guide on How to Expand Managed disk in Azure
```
Comment (feeblefakie) [inline docs/HowToExpandAzureManagedDisk.md:1]:
```suggestion
# A Guide on How to Expand Managed Disk in Azure
```

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Comment by tei-k** [comment]
🙈 Thanks so much!

**Pull request merged**